testsuite/integration-arquillian/tests/other/webauthn/pom.xml

```diff
@@ -13,38 +13,18 @@
     <name>WebAuthn tests</name>
 
     <properties>
-        <selenium.version>4.1.0</selenium.version>
-        <graphene.webdriver.version>3.0.0-alpha.3</graphene.webdriver.version>
-        <htmlunit.driver.version>3.58.0</htmlunit.driver.version>
-        <arquillian.drone.version>3.0.0-alpha.4</arquillian.drone.version>
         <firefoxUserPreferences>${project.build.directory}/dependency/firefox-user-preferences.js
```

Comment:
Not sure why this suddenly has a newline before the end, but can likely be removed.

Reply:
Probably can be removed - follow-up issue on the WebAuthn tests is here: https://github.com/keycloak/keycloak/issues/29780